integration/spark/src/main/scala/org/apache/spark/sql/execution/strategy/CarbonDataSourceScan.scala

```diff
@@ -169,7 +128,7 @@ case class CarbonDataSourceScan(
   override protected def doCanonicalize(): CarbonDataSourceScan = {
     CarbonDataSourceScan(
       relation,
-      output.map(QueryPlan.normalizeExprId(_, output)),
+      a,
```

Comment:
please rename to the proper variable name like `outputAttibutesAfterNormalizingExpressionIds`

Reply:
done